documentation/automating_windows_targets.md

```diff
@@ -24,9 +24,9 @@ performs common task of restarting a service on demand. The process involves
 these steps:
 
 1.  Run your PowerShell script on a Windows target.
-1.  Create an inventory file to store information about the target.
-1.  Convert your script to a task.
-1.  Execute your new task.
+2.  Create an inventory file to store information about the target.
```

Comment:
? I like the `1.` because you don't have to modify the other items if we add a new step in the middle of the list

Reply:
Oh, this is a VS Code extension doing this when I save. I also prefer just using 1. I'll see if I can disable this functionality.

Reply:
Found the setting that was doing this 😉